react-state/hannah-app/src/ToggleButton.jsx

```diff
@@ -0,0 +1,28 @@
+import { useState } from 'react';
+
+export default function ToggleButton({ text, color }) {
+  const [isClicked, setClick] = useState(false);
+  console.log('useState ', isClicked);
+  const buttonStyle = {
+    backgroundColor: color,
+  };
+
+  function handleClick() {
+    console.log('before setter ', isClicked);
+    setClick(!isClicked);
+    console.log('after setter ', isClicked);
+  }
+  return (
+    <button
+      onClick={handleClick}
+      style={
+        isClicked
+          ? buttonStyle
+          : {
+              backgroundColor: color,
```

Comment:
You get the same background color whether or not the button has been clicked. Try `white` if it's not clicked, that way its color will toggle.